generators/client/templates/vue/package.json.ejs

```diff
@@ -27,25 +27,25 @@ limitations under the License.
     "node_modules"
   ],
   "dependencies": {
-    "@fortawesome/fontawesome-svg-core": "1.2.12",
-    "@fortawesome/free-solid-svg-icons": "5.6.3",
-    "@fortawesome/vue-fontawesome": "0.1.4",
+    "@fortawesome/fontawesome-svg-core": "1.2.15",
+    "@fortawesome/free-solid-svg-icons": "5.7.2",
+    "@fortawesome/vue-fontawesome": "0.1.5",
     "axios": "0.18.0",
-    "bootstrap": "4.1.3",
+    "bootstrap": "4.3.1",
     "bootstrap-vue": "2.0.0-rc.11",
```

Comment:
Should we also upgrade to `rc.14`?

Reply:
At first, I tried to upgrade bootstrap-vue too, but there is still bugs in it (drop-down not automatically closed, ...). I'd prefer to stick with current working version for now. 
But do not worry, I'll check for their next releases